k8s/namespaces/rd/rd-profile-sync-integration/rd-profile-sync-integration.yaml

```diff
@@ -12,4 +12,4 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/profile-sync:prod-fc75985-20210823143656
+      image: hmctspublic.azurecr.io/rd/profile-sync:prod-fc75985-20210823143656   #{"$imagepolicy": "flux-system:rd-profile-sync-integration"}
```

Comment:
Can you change this policy to `flux-system:rd-profile-sync`? It looks to be using the same, I would remove the image repo/policy for `flux-system:rd-profile-sync-integration`

Reply:
Image policy/repo now updated.